docs/contributing/tasks.md

```diff
@@ -28,19 +28,23 @@ npm scripts are defined in `package.json`. These trigger a number of Gulp tasks.
 
 **`npm run heroku` runs on Heroku build/PR and it will:**
 
-- run `npm run build:compile`
+- run `npm run build:app`
 - start up Express
 
-**`npm run build:compile` will do the following:**
+**`npm run build:app` will do the following:**
 
+- clean the `./public` folder
 - output files into `./public`, or another location via the `--destination` flag
 - copy fonts and images
-- compile JavaScript and Sass, including documentation
+- run sub tasks from `gulp styles` without ESLint code quality checks
+- run sub tasks from `gulp scripts` without StyleLint code quality checks
```

Comment:
Are these the wrong way round?

```suggestion
- run sub tasks from `gulp styles` without StyleLint code quality checks
- run sub tasks from `gulp scripts` without ESLint code quality checks
```

Reply:
@36degrees Well spotted, they've been switched since https://github.com/alphagov/govuk-frontend/commit/4c11f65570ccc438a73d44de22016bd89f79e394 🤦‍♂️ 

✅ Fixed